packages/icons/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sumup/icons",
-  "version": "2.5.0",
+  "version": "2.6.0-canary.0",
```

Comment:
Hm, since the icons won't have a breaking change, I think that we can actually remove the changeset related to icons from the prerelease channel

Reply:
I imagine that we would just need to make sure that when we merge canary into main at the very end, the icons changelog is not overridden